src/extras/global.js

```diff
@@ -10,18 +10,21 @@
   var firstGlobalProp, secondGlobalProp, lastGlobalProp;
   function getGlobalProp () {
     var cnt = 0;
-    var lastProp;
+    var foundLastProp, result;
     for (var p in global) {
       // do not check frames cause it could be removed during import
       if (shouldSkipProperty(p))
         continue;
       if (cnt === 0 && p !== firstGlobalProp || cnt === 1 && p !== secondGlobalProp)
         return p;
+      if (foundLastProp && !result)
+        result = p;
+      if (foundLastProp && !shouldSkipProperty(p))
+        lastGlobalProp = p;
```

Comment:
Assigning to `result` needs to take into account the `shouldSkipProperty` mechanism.

So perhaps:

```suggestion
      if (foundLastProp && !shouldSkipProperty(p)) {
        lastGlobalProp = p;
        result = result || p;
      }
```

The `||` makes sense above because we are explicitly using the JS rough falsy logic.

Reply:
👍  done